### PR TITLE
astro: update checksum

### PR DESCRIPTION
**Pull request by neel-astro**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----
Changes:
- Added `livecheck` section in `astro` Formula to `git_latest`, in order to avoid pre-release being picked up during regular updates
- Updated the SHA256 of the release

### CODE REVIEW

**Comment by chenrui333** [comment]
@neel-astro when is going to be the next stable release? I am ok with adding the livecheck, but decreasing the version might require special ops.

**Comment by neel-astro** [comment]
@chenrui333 we should be able to get the next stable release out by today, so will wait on that in order to avoid decreasing the version

**Comment by neel-astro** [comment]
@chenrui333 has updated 1.3.0 to stable release, although the SHA has changed since it was in pre-release to being a stable release. Not sure how we can override the check for changing SHA and the URL not changing?

**Comment by neel-astro** [comment]
> @chenrui333 has updated 1.3.0 to stable release, although the SHA has changed since it was in pre-release to being a stable release. Not sure how we can override the check for changing SHA and the URL not changing?

@SMillerDev @chenrui333 @carlocab 
Any way to update the SHA without updating the URL for the release, as the release has been changed while going from pre-release to the stable release. Looking for a way to do so, since this is impacting our users as they are downloading older/unstable code for the release.

File: Formula/astro.rb [hunk 1]
```diff
@@ -2,9 +2,14 @@ class Astro < Formula
   desc "To build and run Airflow DAGs locally and interact with the Astronomer API"
   homepage "https://www.astronomer.io/"
   url "https://github.com/astronomer/astro-cli/archive/refs/tags/v1.3.0.tar.gz"
-  sha256 "578410a9dcda1797ff223601317afbc636ca546d6de4998c899e3e347b229be4"
+  sha256 "349d1411950d323de0e5af166444c0abdab77ba22bf5c7a3749cc277ab4791d9"
   license "Apache-2.0"
```
Comment (carlocab) [inline Formula/astro.rb:6]: This won't fix the error, but at least it makes sure your users get the newer version with `brew upgrade`.
```suggestion
  sha256 "349d1411950d323de0e5af166444c0abdab77ba22bf5c7a3749cc277ab4791d9"
  revision 1
  license "Apache-2.0"
```

**Comment by carlocab** [comment]
We can ignore the CI error. Just add a `revision` to the formula so users get the updated version when they do `brew upgrade`.

**Review by carlocab — APPROVED** [review]

**Comment by neel-astro** [comment]
Anything else required in order to merge these changes?

**Comment by neel-astro** [comment]
@carlocab @chenrui333 @SMillerDev thanks for helping out on this one, really appreciate all the help

**Pull request closed without merging**